### ISSUE TITLE
Stale state for new convo

### ISSUE THREAD

**Issue by SySagar** [un/inbox#699]
Suppose we write a new convo and then we want to cancel it then we are not able to do it. Apart fomr this suppose we just hit to different page and again try new convo rather than generating a new empty mail template it populates it with the stale data.

https://github.com/user-attachments/assets/5806ab7c-9691-424c-ba90-f39565ff88e2

**Comment by BlankParticle** [comment]
It is called a draft

**Comment by SySagar** [comment]
Yup I was able to clear draft. Howsoever the stale data should be removed whenver we are interuptting or cancelling it(either by going to new button click or new page etc) or whenever we are hitting the new convo button. 

Can we rather have a save draft section to handle the drafts part?

**Comment by BlankParticle** [comment]
no drafts are auto saved for better user experience

**Comment by BlankParticle** [comment]
if you are talking about saving drafts in a different list other than at `/new` page, we might look into that

**Comment by SySagar** [comment]
> if you are talking about saving drafts in a different list other than at `/new` page, we might look into that

yup that works too.
